pkg/applicationserver/config.go

```diff
@@ -201,6 +202,9 @@ func (c ApplicationPackagesConfig) NewApplicationPackages(ctx context.Context, s
 	// Initialize LoRa Cloud Device Management v1 package handler
 	handlers[loraclouddevicemanagementv1.PackageName] = loraclouddevicemanagementv1.New(server, c.Registry)
 
+	// Initialize LoRa Cloud Geolocation v3 package handler
```

Comment:
Nit; IMO the comment is superfluous considering the nice naming for the package.

Reply:
It's for consistency - my initial commit deleted the comment above as well 🤭.